Sources/Sentry/SentryScope.m

```diff
@@ -134,16 +137,35 @@ - (void)applyToEvent:(SentryEvent *)event {
         event.infoDict = [[NSBundle mainBundle] infoDictionary];
     }
 
+    event = [self callEventProcessors:event];
+    if (nil == event) {
+        return nil;
+    }
+
     // add context to event.context.otherContexts
     NSMutableDictionary *newOtherContexts = [[NSMutableDictionary alloc] initWithDictionary:event.context.otherContexts];
     [newOtherContexts addEntriesFromDictionary:self.context];
     event.context.otherContexts = [newOtherContexts copy];
+    return event;
 }
 
 - (void)setContextValue:(id)value forKey:(NSString *)key {
     [self.context setValue:value forKey:key];
 }
 
+- (SentryEvent *)callEventProcessors:(SentryEvent *)event {
+    NSLog(@"callEventProcessors");
+    SentryEvent *newEvent = event;
+
+    NSLog(@"SentryGlobalEventProcessor.shared.processors %d", SentryGlobalEventProcessor.shared.processors.count);
```

Comment:
Remove `NSLog`, we might want to use `SentryLog` for things that make sense to debug for the user.
In JS also, for example, we tell if an `eventProcessor` returned `null` so we print it we exit the loop early.

Reply:
done